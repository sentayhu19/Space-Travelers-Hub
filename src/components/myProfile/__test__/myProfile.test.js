import React from 'react';
import '@testing-library/jest-dom/extend-expect';
import { render } from '@testing-library/react';
import { Provider } from 'react-redux';
import store from '../../../redux/configureStore';

describe('Test the myprofile Component', () => {
  it('myprofile section renders', () => {
    const { container } = render(
      <Provider store={store}>
        <myprofile />
      </Provider>,
    );

    expect(container).toMatchSnapshot();
  });
});
