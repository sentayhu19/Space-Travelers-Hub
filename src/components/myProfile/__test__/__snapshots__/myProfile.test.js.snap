// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test the Rockets Component rockets section renders 1`] = `
<div>
  <myprofile />
</div>
`;

exports[`Test the myprofile Component myprofile section renders 1`] = `
<div>
  <myprofile />
</div>
`;

exports[`Test the myprofile Component rockets section renders 1`] = `
<div>
  <myprofile />
</div>
`;
